### DISCUSSION TITLE
NGINX Proxy Manager LXC - add geoip2 module

### DISCUSSION THREAD

**Discussion by lepo80** [community-scripts/ProxmoxVE#774] [Ideas]
Is it possible to add the geoip2 module into the NGINX Proxy Manager LXC installation script?

The GeoIP2 module for NGINX is an advanced feature used to determine the geographical location of users accessing your server. It integrates with MaxMind's GeoIP2 databases to map IP addresses to specific data, such as country, city, latitude, longitude, ISP, and more. This information can be utilized to tailor content, apply location-based restrictions, or enhance logging and analytics. The module requires the GeoIP2 library and databases to be installed separately and supports configuration through directives in the NGINX configuration file. It's a modern alternative to the older GeoIP module, offering better accuracy and more detailed data.

Thanks in advance for your feedback!

**Comment by michelroegl-brunner** [comment]
In my opinion i would not install any plugins with the lxc. If anyone needs they can install it themself.

**Reply by lepo80** [reply]
Interesting. Why do you avoid installing a simple module on an LXC?
Actually, I did install it later, but unfortunately, the module stops working after every upgrade. As a result, I'm forced to migrate from a clean installation and reinstall the module each time.
The benefits of having segregation by geolocation are significant, but that's just my opinion.

**Reply by gabriellxxix** [reply]
Hello,

I would like to install the nginx-module-vts in an NPM (Nginx Proxy Manager) setup created using the community script. Could you please guide me on how to accomplish this?

Thank you for your help!

**Reply by tremor021** [reply]
> Interesting. Why do you avoid installing a simple module on an LXC? Actually, I did install it later, but unfortunately, the module stops working after every upgrade. As a result, I'm forced to migrate from a clean installation and reinstall the module each time.

you literaly gave an answer to your own question...

**Reply by lepo80** [reply]
> 
> you literaly gave an answer to your own question...
Really? In this case, why not include the module installation in current script, as optional choice?

**Reply by tremor021** [reply]
Not sure how are you not getting this... We won't install optional stuff because when it breaks because of app update or w/e,  people come to complain to US instead to the app devs. If you wanna use optional stuff, install it yourself and maintain it yourself.
